.changeset/mean-toys-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-common': patch
+---
+
+Adds config option to set ownership for newly created schemas and tables in Postgres
```

Comment:
Probably want to update this to include the new `role` naming and maybe some sample config too? :pray: